### PR TITLE
Update At A Glance Dashboard 

### PR DESCRIPTION
**Pull request by dereksmart**
Fixes # 4020

This updates the specs of the dashboard to match the new designs/information in p6TEKc-xF-p2. 

There are still a couple of kinks that need to be worked out in this, such as: 

- [x] more DRYness when showing the inactive/uninstalled state of the plugins
- [x] direct link from the SectionHeader `activate` and `install` links/buttons
- [x] The `loading` card for Backups is not 50%
- [x] Inactive stats module needs it's own new component (see screenshots in the p6TEKc-xF-p2)
- [x] `Active` text in the SectionHeaders for Pro cards need styling

**To test:** 

Non-pro: 
- activate/deactivate non-pro modules.  Make sure data displays correctly. 

Akismet: 
- Active and installed 
- Active and installed with no API key 
- Inactive and installed
- Uninstalled

Scan: 
- VaultPress Active and installed 
- VaultPress Active and installed with spoofing a threat.  You must hardcode the value in place of this `const threats = this.props.getScanThreats();` in `scan.jsx`
- VP inactive but installed
- VP uninstalled

Backups: 
- VaultPress Active and installed 
- VP inactive but installed
- VP uninstalled

### CODE REVIEW

File: _inc/client/at-a-glance/plugins.jsx [hunk 1]
```diff
@@ -46,19 +50,30 @@ const DashPluginUpdates = React.createClass( {
...
-								__( '%(number)s plugin needs updating.', '%(number)s plugins need updating.', {
-									count: pluginUpdates.count,
-									args: {
-										number: pluginUpdates.count
-									}
-								} )
-							}
-						</strong>
-						<br/>
+						{
```
Comment (MichaelArestad) [inline _inc/client/at-a-glance/plugins.jsx]: @dereksmart When you get a minute, can you check to make sure I did this section right? (I don't fully understand it, but I guessed based on your work and it works)

File: _inc/client/components/dash-item/style.scss [hunk 2]
```diff
@@ -69,7 +69,7 @@
 }
 .jp-dash-item__active-label {
 	display: inline-block;
-	padding: rem( 7px ) 0;
```
Comment (MichaelArestad) [inline _inc/client/components/dash-item/style.scss]: It was set this way to line up with the buttons (same size, weight, padding). Haven't tested your change yet, though.

**Comment by jeffgolenski** [comment]
@dereksmart I'm seeing an issue with the activation / deactivation of Akismet. Shows active on the dash, but inactive within settings, and the toggle isn't working. 

akismet-deactivate-issue


I'm assuming this isn't going to be an issue moving forward since the settings areas of our pro features won't have toggles?

example:
premium feature states - 02


Also: don't pay attention to the colors in this gif, licecap doesn't record them correctly.

File: _inc/client/components/dash-item/index.jsx [hunk 3]
```diff
@@ -3,86 +3,179 @@
...
+				>
+					{ this.props.statusText }
+				</SimpleNotice>;
 				break;
 			case 'is-working':
-				icon = 'checkmark';
-				break;
-			case 'is-premium-inactive':
-				icon = 'lock';
+				status = <span className="jp-dash-item__active-label">Active</span>;
```
Comment (eliorivero) [inline _inc/client/components/dash-item/index.jsx]: @dereksmart perhaps this should be translatable?

File: _inc/client/components/dash-item/index.jsx [hunk 4]
```diff
@@ -3,86 +3,179 @@
...
+					slug={ this.props.module }
+					activated={ this.props.isModuleActivated( this.props.module ) }
+					toggling={ this.props.isTogglingModule( this.props.module ) }
+					toggleModule={ this.props.toggleModule }
+				    compact={ true }
+				/>
 			);
+
+			if ( 'manage' === this.props.module && 'is-warning' === this.props.status ) {
+				let text = __( 'Updates Needed' );
```
Comment (eliorivero) [inline _inc/client/components/dash-item/index.jsx]: Redundant variable, the function call can be directly placed in line 126

File: _inc/client/components/dash-item/index.jsx (hunk 3 above)
Comment (dereksmart) [inline _inc/client/components/dash-item/index.jsx]: good catch

**Pull request merged**